### PR TITLE
Feat: completed archives, rearranged folder structure, added dark mode

### PR DESCRIPTION
**Pull request by vinc3nati**
- Added Dark Mode Switcher
- Added Account Dropdown
- Completed Archive functionality

image

image

image

image

### CODE REVIEW

File: src/components/NoteCard/NoteCard.jsx [hunk 1]
```diff
@@ -1,20 +1,111 @@
...
+      });
+      setDisabled(false);
+    }
+  };
+
+  const handleDeleteArchives = async () => {
+    setDisabled(true);
+    const response = await deleteArchives({ notesId: _id, token });
+    if (response.data.archives) {
+      console.log(response.data.archives);
```
Comment (anuj-kumary) [inline src/components/NoteCard/NoteCard.jsx]: Remove console.log

**Review by PeeyushPrashant — APPROVED** [review]

**Review by anuj-kumary — APPROVED** [review]
Your code is looking good to merge
I Really like:

- Your code is properly formatted and has a consistent indentation
- Code is modular and easier to understand

File: src/components/NoteCard/NoteCard.jsx (hunk 1 above)
Comment (vinc3nati) [reply src/components/NoteCard/NoteCard.jsx]: sure

File: src/App.js [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
 import Mockman from "mockman-js";
-import "./App.css";
 import { Home } from "./pages/Home/Home";
 import { Footer } from "./components/Footer/Footer";
 import { Auth } from "./pages/Auth/Auth";
 import { useData } from "./contexts/data-context";
 import { Loader } from "./components/Loader/Loader";
 import { Notes } from "./pages/Notes/Notes";
 import { PrivateRoute } from "./components/PrivateRoute/PrivateRoute";
+import { Archives } from "./pages/Archives/Archives";
```
Comment (ramalingaa) [inline src/App.js:10]: Try using index imports,it will reduce the no. of imports

**Review by lomesshh — APPROVED** [review]
Clean code
It's great that you have implemented dark mode
looks good to merge

**Review by ramalingaa — APPROVED** [review]
Code layout is properly aligned and clean. 
PR can be merged

**Pull request merged**